components/leaderboard/Leaderboard.module.scss

```
@@ -11,17 +11,201 @@
+
+.networkFilters .contentDropdown button {
+  align-items: center;
+  display: inline-flex;
+  gap: 0.35rem;
+  line-height: 1.1;
+}
+
+.dropdownCaret {
+  border-left: 0.28em solid transparent;
+  border-right: 0.28em solid transparent;
+  border-top: 0.28em solid currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)